File: Java/Lista_03(Estrutura_Repetitiva)/Dentro_Fora.java
import java.util.Scanner;

public class Dentro_Fora {

	public static void main(String[] args) {

		Scanner s = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n = s.nextInt();
		
		int contD = 0, contF = 0;
		
		for(int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			int x = s.nextInt();
			if(x<10 || x>20) {
				contF++;
			}
			else {
				contD++;
			}
		}
		
		System.out.println(contD + " DENTRO");
		System.out.print(contF + " FORA");
		
		s.close();
		
	}

}

File: C++/Lista_03(Estrutura_Repetitiva)/Dentro_Fora.cpp
#include <iostream>

using namespace std;

int main(){

    int n, x, contD=0, contF=0;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    for(int i=0;i<n;i++){
        cout << "Digite um numero: ";
        cin >> x;
        if(x<10 || x>20){
            contF++;
        }
        else{
            contD++;
        }
    }

    cout << contD << " DENTRO" << endl;
    cout << contF << " FORA";

    return 0;

}